### DISCUSSION TITLE
Problem with certain usernames

### DISCUSSION THREAD

**Discussion by ninto1** [sherlock-project/sherlock#993] [Q&A]
Hi.
I recently wanted to use sherlock to try and find a user with - at the beginning of the username. 
Sherlock gave me an error, because it thought I want to use a command.

Pls help.
Thx in advance (^-^)

**Comment by Kataphractoi** [comment]
you can put it in quotes like "-[username]" python will recognize things inside quotes as strings only

**Comment by Kataphractoi** [comment]
though I have problems with multilingual usernames like ones in chinese or have english and another language mixed in. I kinda wish they added multilingual support

**Reply by SethFalco** [reply]
Could you share some examples of websites that allow such characters?

**Comment by Cfomodz** [comment]
herlock interprets usernames starting with `-` as command-line options, causing errors.

Along with the answer above suggesting you can wrap the username in quotes:

You can also use `--` to separate options from arguments:

```bash
sherlock -- -username
```
